installer/api/handlers_latest_release.go

```
@@ -1,31 +1,55 @@
 package api
 
 import (
-	"io"
+	"fmt"
 	"net/http"
+	"strings"
 
 	"github.com/coreos/tectonic-installer/installer/version"
+	"golang.org/x/net/html"
 )
 
 // latestReleaseHandler gets the tectonic's latest release from coreos.com.
-func latestReleaseHandler(w http.ResponseWriter, req *http.Request, _ *Context) error {
-	res, err := http.Get("https://coreos.com/tectonic/api/releases/latest")
+func fetchLatestRelease() string {
+	var rel string
+	response, err := http.Get("https://coreos.com/tectonic/api/releases/latest/")
 	if err != nil {
-		return newBadRequestError("Failed to get a response from coreos.com: %s", err)
+		fmt.Printf("Failed to get a response from coreos.com: %s", err)
```

Just logging an error with “fmt.Printf” is not really sufficient. We should return an error here.